src/olympia/yara/admin.py

```diff
@@ -0,0 +1,54 @@
+import json
+
+from django.contrib import admin
+from django.utils.html import format_html
+
+from olympia import amo
+from olympia.amo.urlresolvers import reverse
+
+from .models import YaraResult
+
+
+@admin.register(YaraResult)
+class YaraResultAdmin(admin.ModelAdmin):
+    actions = None
+
```

Comment:
Add `view_on_site = False` as well (I guarantee you that's the first thing QA will try :stuck_out_tongue: )